aspnetcore/blazor/globalization-localization.md

```diff
@@ -873,6 +873,53 @@ To further understand how the Blazor framework processes localization, see the [
 
 :::zone-end
 
+## Shared resources
+
+To create localization shared resources, adopt the following approach. For additional information, see the shared resources guidance in the <xref:fundamentals/localization#make-the-apps-content-localizable> article.
+
+* Create a dummy class with an arbitrary class name. In the following example:
+
+  * The app uses the `BlazorSample` namespace, and localization assets use the `BlazorSample.Localization` namespace.
+  * The dummy class is named `ComponentLocalization`.
+  * The class file is placed in a `Localization` folder at the root of the app.
+
+  `Localization/ComponentLocalization.cs`:
+
+  ```csharp
+  namespace BlazorSample.Localization
+  {
+      public class ComponentLocalization
```

Comment:
`SharedResource` might be a better name, coz it could be used for components and other things